### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by gedewon**
## PR checklist;

- [x] Parse Zeplin's design.
- [x] Use the information detailed in Zeplin to style the website.
- [x]  final design matches Zeplin's design.

### CODE REVIEW

File: src/component/Layout/DefaultLayout.module.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+}
+
+header {
+  display: flex;
+  align-items: center;
+}
```
Comment (roshan-bajgain) [inline src/component/Layout/DefaultLayout.module.css:106]: - [ ] Your `Bookstore CMC` should be on the same line and there should be a logo on the right side of the page. 
Kindly implement it

- [ ] Books and categories link has different color kindly implement it

### Your design
hiss
### Figma design
figma

File: src/component/Layout/DefaultLayout.module.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  height: 0.125rem;
+  margin: 5.875rem 0 2.313rem;
+  border: solid 1px #e8e8e8;
+}
+
+.headerComponent {
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
```
Comment (roshan-bajgain) [inline src/component/Layout/DefaultLayout.module.css:100]: - [ ] Due to greater width it is horizontally overflowing and the height of the content is bigger kindly fix it.
- [ ] Circularprogress bar should be in the middle of the page. Kindly make changes see the image below:
### Your design
hiss

File: src/component/InputBook/InputBook.module.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+footer {
+  padding: 0 95px;
+  height: 291px;
+}
```
Comment (roshan-bajgain) [inline src/component/InputBook/InputBook.module.css:65]: - [ ] The footer should be aligned in the same line as shown in the design. Please make changes to it
###  Figma design
figma
### Your design

your

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Gedewon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- [ ] Kindly provide credentials to [Zeplin](https://zeplin.io/) in README file

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @Gedewon, 

Well done making the changes required by the previous reviewer 🙌
Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**